tests/test_particle_sets.py

```diff
@@ -54,6 +54,27 @@ class MyParticle(ptype[mode]):
     assert np.allclose([p.v for p in pset], v_vals, rtol=1e-12)
 
 
+@pytest.mark.parametrize('mode', ['scipy', 'jit'])
+def test_pset_create_fromparticlefile(fieldset, mode, tmpdir):
+    filename = tmpdir.join("pset_fromparticlefile.nc")
+    lon = np.linspace(0, 1, 10, dtype=np.float32)
+    lat = np.linspace(1, 0, 10, dtype=np.float32)
+    pset = ParticleSet(fieldset, lon=lon, lat=lat, pclass=ptype[mode])
+    pfile = pset.ParticleFile(filename, outputdt=1)
+
+    def DeleteLast(particle, fieldset, time):
+        if particle.lon == 1.:
+            particle.delete()
```

Comment:
So you wanted to check also deleted particles are written, right?

Reply:
Yes correct